measurements/podvm-measure.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
```

Comment:
nit: you might want to put this under the scripts folder and switch to absolute paths instead of relative path in the run_measurements.sh script